### ISSUE TITLE
Further considerations from crypto trading platforms with leverage

### ISSUE THREAD

**Issue by jonathanbier** [bitcoin-cap/bcap#44]
In the stakeholder section, for economic nodes, you could consider adding the following bullet points under incentives:

•	**Comply with requests from large customers**
•	**Ensure certainty as to the value of Bitcoin used as margin and collateral for any client positions, at all times.** 

The first point was quite important during the blocksize war, many exchanges just wanted to keep customers happy. 

Another important consideration during the blocksize war, is that many of the trading platforms which supported leverage and complex derivatives were more often on the “smaller block side” (e.g. Bitfinex and BitMEX), while platforms offering just spot tended to be larger blockers (Coinbase). There were obviously several factors driving this, but one factor is that the use of Bitcoin as a margin currency for highly leveraged positions meant it was more obvious to some platforms why they could not adopt the less certain “most work no matter the rules” chain is Bitcoin idea. While for Coinbase offering spot, this “most work” idea and a short period of uncertainty while miners worked it out, seemed more feasible.

**Comment by moneyball** [comment]
Thank you @jonathanbier! I agree the first point should be added. @lynalden, @rencryptofish, others, what do you think?

Can you elaborate on your 2nd point? I'm not sure I understand what margin platforms/services did or would support if not the "most work"?

**Comment by lynalden** [comment]
> Thank you @jonathanbier! I agree the first point should be added. @lynalden, @rencryptofish, others, what do you think?
> 
> Can you elaborate on your 2nd point? I'm not sure I understand what margin platforms/services did or would support if not the "most work"?

I agree with adding the first point, and seeking more clarification on the rest.

**Comment by jonathanbier** [comment]
> Can you elaborate on your 2nd point? I'm not sure I understand what margin platforms/services did or would support if not the "most work"?

Well during the blocksize war Bitfinex said they would support the "incumbent chain" and call that coin Bitcoin, whatever that means. BitMEX said "BitMEX considers any and all contentious hardfork tokens as altcoins". These were the largest two deriv platforms at the time.

On the other hand, Coinbase, the largest spot venue I think, said it would call the "most work" chain Bitcoin, in the event of a split. Again not entirely sure what that means.

In my view, the spot vs deriv distinction was an important driver of these policies.  Maybe Coinbase had some idea that a hardfork chain split would happen and the COinbase exchange BTC market would be shit down for a while, maybe a few hours, while the miners sorted it out.

Bitfinex and BitMEX needed to be up 24*7. They were deriv platfoms with large open interest and leverage. In the case of BitMEX, average leverage of around 12x. Suspending everything would not be easy. With Bitfinex, many people had Bitcoin loans. There is a risk that a contentious hardfork and an approach of "wait and see what the miners do", coudl result in a chaotic unwinding of the open leveraged positions and cascading liquidations. This would be bad for business. So they had a different approach to Coinbase.
